sungjin/.env.sample

```
@@ -0,0 +1 @@
+PORT = YOUR PORT NUMBER SHOULD BE HERE! 
```

`app.js` 파일에서 서버를 구동하기 위해 필요한 환경변수가 sample 파일에서 모두 확인이 불가능한 것 같습니다!

다른 환경변수들도 어떤 값이 할당되는지 확인할 수 있도록 `.env.sample` 파일을 수정해주세요!

```suggestion
DB_CONNECTION = mysql
DB_HOST = HOST
DB_USERNAME = USERNAME
DB_PASSWORD =  PASSWORD
DB_DATABASE = DB_NAME 
DB_PORT = 3306

DATABASE_URL="mysql://username:password@127.0.0.1:3306/database_name"

PORT = PORT
```